src/RockBridge.cpp

```diff
@@ -273,14 +273,17 @@ void RockBridge::instantiatePluginComponents(sdf::ElementPtr modelElement, Model
 {
     sdf::ElementPtr pluginElement = modelElement->GetElement("plugin");
     while(pluginElement) {
-        string pluginFilename = pluginElement->Get<string>("filename");
-        gzmsg << "RockBridge: found plugin "<< pluginFilename << endl;
+        string filename = pluginElement->Get<string>("filename");
+        string name = pluginElement->Get<string>("name");
+        gzmsg << "RockBridge: found plugin name='" << name << "' "
+              << "filename='" << filename << "'" << endl;
+
         // Add more model plugins testing them here
         if(pluginFilename == "libgazebo_thruster.so")
```

Comment:
`if(filename == "libgazebo_thruster.so")`

Reply:
Thanks. Sorry for the compilation error, this was basically extracted from another PR (#26)